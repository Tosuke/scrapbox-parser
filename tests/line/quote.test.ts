/* global describe it expect */

describe('quote', () => {
  it('Simple quote', () => {
    expect('> Simple quote').toMatchSnapshotWhenParsing()
  })

  it('Empty quote', () => {
    expect('>').toMatchSnapshotWhenParsing()
  })
})
