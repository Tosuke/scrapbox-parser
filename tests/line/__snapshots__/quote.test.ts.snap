// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quote Empty quote: toMatchSnapshotWhenParsing 1`] = `
Array [
  Object {
    "indent": 0,
    "nodes": Array [
      Object {
        "nodes": Array [],
        "type": "quote",
      },
    ],
    "type": "line",
  },
]
`;

exports[`quote Simple quote: toMatchSnapshotWhenParsing 1`] = `
Array [
  Object {
    "indent": 0,
    "nodes": Array [
      Object {
        "nodes": Array [
          Object {
            "text": " Simple quote",
            "type": "plain",
          },
        ],
        "type": "quote",
      },
    ],
    "type": "line",
  },
]
`;
